src/components/ArticlePoster.jsx

```diff
@@ -20,7 +21,8 @@ const ArticlePoster = () => {
   }, []);
 
   if (loading)  return <p>loading...</p>
-  return <section className="article-poster">
+  return <>
+  <section className="article-poster">
```

Comment:
I'm glad this isn't just a div, however is section a good description? In our case it literally is an article so why not use that?